intelmq/bots/collectors/xmpp/collector.py

```diff
@@ -81,39 +85,90 @@ def init(self):
             self.logger.error('Could not import sleekxmpp. Please install it.')
             self.stop()
 
+        # Retrieve Parameters from configuration
+        xmpp_user = getattr(self.parameters, "xmpp_user", None)
+        xmpp_server = getattr(self.parameters, "xmpp_server", None)
+        xmpp_password = getattr(self.parameters, "xmpp_password", None)
+
+        if None in (xmpp_user, xmpp_server, xmpp_password):
+            self.logger.error('No User / Password provided')
+            self.stop()
+        else:
+            xmpp_login = xmpp_user + '@' + xmpp_server
+
+        self.userlist = getattr(self.parameters, "xmpp_userlist", [])
+        # When configured in manager this is most likely a ,-separated string, we'd like an array
+        if type(self.userlist) is str:
+            self.userlist = [u.strip() for u in self.userlist.split(",")]
+
+        self.whitelist_mode = getattr(self.parameters, "xmpp_whitelist_mode", False)
+
+        self.muc = getattr(self.parameters, "use_muc", None)
+        xmpp_room = getattr(self.parameters, "xmpp_room", None) if self.muc else None
+        xmpp_room_nick = getattr(self.parameters, "xmpp_room_nick", None) if self.muc else None
+        xmpp_room_password = getattr(self.parameters, "xmpp_room_password", None) if self.muc else None
+
+        self.full_xml = getattr(self.parameters, "pass_full_xml", None)
```

Comment:
Because the Login-Name might differ from the Room name. In this way you can use one account within the same room twice or more AND distinguish the bots.